vscode/src/gh-copilot/tools.ts

```diff
@@ -116,15 +117,14 @@ function tool(
 }
 
 async function invokeTool(
+  context: vscode.ExtensionContext,
   input: any,
   toolFn: (input: any) => Promise<any>,
 ): Promise<vscode.LanguageModelToolResult> {
-  log.info("Invoking tool");
+  updateCopilotInstructions("ChatToolCall", context);
```

Comment:
Am I reading this right that was call this on every tool invocation now, and if not configured already we always prompt the user?

Reply:
Correct

Reply:
Fixed the "don't show again" handling. 

Will try working this _into_ the chat now as we discussed, but separately from this PR